query-engine/src/start.py

```
@@ -0,0 +1,63 @@
+import asyncio
+import uvicorn
```

Ruff (0.8.2)

`uvicorn` imported but unused

Remove unused import: `uvicorn`

(F401)